cmd/humioctl/filter_alerts_list.go

```diff
@@ -0,0 +1,63 @@
+// Copyright © 2020 Humio Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"strings"
+
+	"github.com/humio/cli/cmd/internal/format"
+	"github.com/spf13/cobra"
+)
+
+func newFilterAlertsListCmd() *cobra.Command {
+	cmd := cobra.Command{
+		Use:   "list [flags] <view>",
+		Short: "List all filter alerts in a view.",
+		Args:  cobra.ExactArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			view := args[0]
+			client := NewApiClient(cmd)
+
+			filterAlerts, err := client.FilterAlerts().List(view)
+			exitOnError(cmd, err, "Error fetching filter alerts")
+
+			actions, err := client.Actions().List(view)
+			exitOnError(cmd, err, "Unable to fetch notifier details")
```

Comment:
"notifier" -> "actions". That was something we renamed a long time ago.

Reply:
Please also update variable names.

Reply:
I can't leave a comment on the correct line (Thanks GitHub).

About the action names from the graphQL: They are available from the graphql endpoint, however we don't map them down to the api/alert struct. So unless we expand that struct we don't have the information here. That would require a action map <id, name> to be appended. Is the favorable to calling the alert api?

Reply:
I guess you mean favorable to calling the *actions* api? It means one less call to the server, as everything can be returned from that one call, so yes, that is more performant.

Also, I think that the alternative is to implement a new method on the action api that only returns part of the action data, as some of it requires another permission, which is not needed to access the alerts api.

Reply:
I want to solve this by adding an extra field called ActionWithNames (ID string, Name string) which doesn't get exported and read from file. I don't like this solution, but I'm struggling a bit finding a cleaner method.

Which fields can't I access on Actions? Right now it's using the `searchDomain(name: $viewName)` query.

Reply:
I was only suggesting that the command prints the alerts, even if `client.Actions().List(view)` fails.

Reply:
> Which fields can't I access on Actions?

The field `yamlTemplate` as well as all fields in the subtypes.